src/warnet/lnnode.py

```diff
@@ -0,0 +1,115 @@
+
+import docker
+import json
+import os
+from docker.models.containers import Container
+from warnet.client import get_file_from_container
+from warnet.utils import (
+    exponential_backoff,
+    generate_ipv4_addr
+)
+
+CONTAINER_PREFIX_LN = "ln"
```

Comment:
```suggestion
CONTAINER_PREFIX_LN = "lnd"
```

Reply:
no because the container could be a CLN node

Reply:
yeah

Reply:
And we won't give these a cutsie prefix like `bolt` or `spark` :'(

Reply:
lets just go back to using Winston Churchill quotes for everything